Service/SchemaService.php

```diff
@@ -0,0 +1,304 @@
+<?php
+
+namespace CommonGateway\CoreBundle\Service;
+
+use App\Entity\Entity;
+use App\Entity\ObjectEntity;
+use App\Entity\Value;
+use Doctrine\ORM\EntityManagerInterface;
+use Monolog\Logger;
+
```

Comment:
Missing a docblock with generic function description and authors here

Reply:
I will add some basic description, but i haven't touched this service, so not 100% sure what comments to put above it